reporting/reporting_miro_transformer/src/transform.py

```diff
@@ -40,7 +38,7 @@ def convert_date_to_iso(date_string):
     try:
         return parse(date_string).date().isoformat()
     except (ValueError, TypeError):
-        return date_string
+        return None
```

Comment:
minor: Good Python practice is to throw an exception rather than return sentinel values, so you're not relying on the caller to check `if foo is None`